src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag

```diff
@@ -64,6 +64,12 @@
                             <f:feedbackSessionPublishButton buttonType="${sessionPanel.feedbackSessionPublishButton.buttonType}"
                                                             publishButton="${sessionPanel.feedbackSessionPublishButton}" />
                         </div>
+                        <br>
+                        <div>
+                            <button id="button_print" type="input" class="btn btn-primary btn-block">
```

Comment:
I think I missed this earlier, but this id should be `button-print` instead, and the js file will need to be updated as well.